### DISCUSSION TITLE
預警地震的優良軟體地牛 wake up

### DISCUSSION THREAD

**Discussion by wangtyphoon** [1101-GS4719-C/Python-Community-GS4719-C#981] [General]
隨著2022年的結束，我們也告別了這許多地震的一年，在過去的這段時間裡，我最喜歡的地震預警軟體絕對是地牛wake up ，簡單的介面，有效的告警系統，成功在台東花蓮一連串地震發生前，把我從床上叫醒。
另外也推薦real time MT 可以知道地震的走向和強度喔!但略有延遲。
https://eew.earthquake.tw/
https://rmt.earth.sinica.edu.tw/

**Comment by swerfg03joe** [comment]
雖然多數的地震預警系統都是利用時間差來做為告示，但有總比沒有好，能快速對即將到來的地震作出相對應的姿勢以減輕傷害，還是比較重要的，只能期盼未來有學者能找出真正的地震預測方法。

**Comment by harrylee1971** [comment]
有這東西真的比較好，可以讓人先預先知道準備有地震出來，差那幾秒可能就是好幾個生命的失去或是存活

**Comment by s96015welson** [comment]
我認為地震警報非常重要，也多虧了手機普及使得地震預警能第一時間告知使用者，不過現在政府也會發布地震預警到手機了，而且還響得滿大聲的，不知道是政府的快還是這幾支程式更快呢?

**Comment by HsuChen88** [comment]
就算地震提前一分鐘知道，我也來不及跑，因為手機根本不會響，我是被地震警報邊緣的人。

**Comment by 110302550** [comment]
地震常常造成很多種危害，若是有這個技術，我們應該更加致力於這項技術，把它做到完善，這樣就能避免掉很多可避免的災難。
